JASP-Engine/JASP/R/graphPackage.R

```diff
@@ -0,0 +1,1088 @@
+.graphOptions <- list2env(list(
```

Comment:
The JASPGraph package can not be added as a package in the JASP framework at the moment, and Mac users can not add package/library in JASP by themselves so Frans suggests this is the easiest way to call the package right now.